common/src/main/java/org/wso2/testgrid/common/util/TestGridUtil.java

```diff
@@ -160,7 +161,10 @@ public static String executeCommand(String command, File workingDirectory, Deplo
      * @return test grid home path
      */
     public static String getTestGridHomePath() throws IOException {
-        String testGridHome = EnvironmentUtil.getSystemVariableValue(TESTGRID_HOME_ENV);
+        String testGridHome = EnvironmentUtil.getSystemVariableValue(TestGridConstants.TESTGRID_HOME_ENV);
+        if (testGridHome == null) {
```

Comment:
Isn't it better to check for Null or Empty both?

Reply:
yes, done.